app/src/main/res/layout/walkthrough_topic_summary_view.xml

```diff
@@ -15,7 +15,7 @@
     android:layout_height="wrap_content"
     android:clickable="true"
     android:elevation="4dp"
-    android:layout_margin="@dimen/space_20dp"
+    android:layout_margin="@dimen/walkthrough_topic_summary_parent_margin"
```

Comment:
Perhaps instead: ``walktrough_topic_summary_container_margin`` (since it's a container; 'parent' is a bit vague).

Reply:
Okay

Reply:
Resolving this on behalf of @BenHenning